website/views/options/social/group.jade

```diff
@@ -51,7 +51,12 @@ a.pull-right.gem-wallet(ng-if='group.type!="party"', popover-trigger='mouseenter
             img.pull-right(ng-show='group.logo', ng-src='{{group.logo}}')
             markdown(text='group.description')
           hr
-          small.muted Group ID: {{group._id}}
+          p=env.t('groupLeader')
+            a(class='badge badge-info', ng-click='clickMember(group.leader, true)')
+              | {{group.leader}}
+          .formatting-help
```

Comment:
What I'm getting at here is that the `formatting-help` class belongs to the markdown formatting section. So it's great that the behavior it creates is identical to what you want, but that may not always be the case. If the markdown formatting section ever gets a major refactor, the styling for `formatting-help` is likely to change in unexpected ways. 

So, yes, while you _can_ use it, I don't think you _should_ use it. 

There are a couple of directions you can go (listed in order of most preferred method to least preferred method):

1) Use existing bootstrap classes to get the behavior you want. You can find them here: http://getbootstrap.com/css/ and here: http://getbootstrap.com/components/

2) Abstract out the name of `formatting-help` to be more descriptive of what it is actually doing. Then it won't be tied specifically to the markdown help section and can be used any place you want.

3) Create a new class that duplicates the styling of `formatting-help`